unittests/adt/test_chunked_vector.cpp

```diff
@@ -214,3 +213,103 @@ TYPED_TEST (CVIterator, Predecrement) {
     EXPECT_EQ (*it, 2);
     EXPECT_EQ (it, this->cv_.begin ());
 }
+
+//  - An underscore '_' indicates uninitialized storage.
+//  - An arrow '->' indicates the list of chunks.
+TEST (ChunkedVectorResize, FillCurrentTailChunk) {
+    cvector_int cv;
+    cv.emplace_back (13);
+    // Before the resize we have a single chunk:
+    //     [ 13, _ ]
+    // After it, we fill the tail chunk with default-initialized int:
+    //    [ 13, 0 ]
+    cv.resize (2U);
+    EXPECT_EQ (cv.size (), 2U);
+    EXPECT_EQ (cv.capacity (), 2U);
+
+    cvector_int::const_iterator it = cv.begin ();
+    EXPECT_EQ (*it, 13) << "Element 0 (chunk 0, index 0)";
+    std::advance (it, 1);
+    EXPECT_EQ (*it, 0) << "Element 0 (chunk 0, index 1)";
+}
+
+TEST (ChunkedVectorResize, FillInitialChunkAndPartialSecond) {
+    cvector_int cv;
+    cv.emplace_back (17);
+    // Before the resize we have a single chunk:
+    //     [ 13, _ ]
+    // Extending this to three members will produce:
+    //     [ 13, 0 ] -> [ 0, _ ]
```

Comment:
Same as above, should it be [**17**, 0]?

Reply:
As above.